frontend/components/cards/ModuleCard.tsx

```diff
@@ -169,10 +169,26 @@ const ModuleCard: FC<ModuleCardProps> = ({
         <CardHeader className={"text-center"}>
           <small dangerouslySetInnerHTML={{ __html: header }} />
         </CardHeader>
-      ) : module.dateString ? (
+      ) : module.title && module.dateString ? ( // if title and dataString
         <CardHeader className={"text-center"}>
+          <small dangerouslySetInnerHTML={{ __html: module.title }} />
+          <br></br>
           <small dangerouslySetInnerHTML={{ __html: module.dateString }} />
         </CardHeader>
+      ) : module.name && module.dateString ? ( // if name and dateString
+        <CardHeader className={"text-center"}>
+          <small dangerouslySetInnerHTML={{ __html: module.name }} />
+          <br></br>
+          <small dangerouslySetInnerHTML={{ __html: module.dateString }} />
+        </CardHeader>
+      ) : module.name ? ( // if just name
+        <CardHeader className={"text-center"}>
+          <small dangerouslySetInnerHTML={{ __html: module.name }} />
+        </CardHeader>
+      ) : module.title ? ( // if just title
+        <CardHeader className={"text-center"}>
+          <small dangerouslySetInnerHTML={{ __html: module.title }} />
+        </CardHeader>
```

Comment:
Unfortunatly, module.name and module.title are not the same. Depending on the type of card, it may have title or name, or neither. But your implementation does look better and cleaner so I will be adding that right now.

Reply:
Do you have an example of this? I thought that all module with `name` also have `title` (e.g. [entities `title` is set to its `name`](https://github.com/ModularHistory/modularhistory/blob/601bebd7088363bc315482ff1d3af35e59a76cd8/apps/entities/models/entity.py#L183)).

Reply:
![image](https://user-images.githubusercontent.com/59071999/132762820-302e11bf-794f-46f8-8f4d-6499a2c3982f.png)
This is using just name.
![image](https://user-images.githubusercontent.com/59071999/132762926-005c0f5f-159d-4821-8a4d-f7651255a490.png)
This is using just title.
![image](https://user-images.githubusercontent.com/59071999/132763022-742ec6c1-6684-4153-8b8a-41d5eab2f1dd.png)
This is using both.

Some have just title and some have just name. I may have been mistaken about some cards having neither.

Reply:
Oh interesting. Entity's `title` is supposed to be set from its `name`, but the `title` field was added after `name` and does not get set until the object is saved. All our current entities have not been re-saved since the `title` field was added, so they don't have `title` values yet. Hopefully that makes sense!

I'll run a script in production shortly to set the `title` values from `name`.

Reply:
I'll check it out now, thanks!